### PR TITLE
[GEN-573] Show GDPR link before calculating price

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Show a link to privacy policy in price calculator before showing price

  - Only show the link on the last step of the price calculator

- Add PageLink function to get privacy policy link

## Justify why they are needed

Be compliant with GDPR

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2600** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2600" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2600?utm_source=stack-comment).

File: apps/store/public/locales/en/purchase-form.json [hunk 1]
```diff
@@ -93,6 +93,7 @@
   "FIELD_SUB_TYPE_APARTMENT_OPTION_RENT": "Rent it",
   "FIELD_YEAR_OF_CONSTRUCTION_LABEL": "Year of construction",
   "FIELD_ZIP_CODE_LABEL": "Postal code",
+  "GDPR_LINK_BEFORE_OFFER": "How we handle personal data*",
```
Comment (gustaveen) [inline apps/store/public/locales/en/purchase-form.json:96]: nit: I suppose the `*` shouldn't be there :)

**Review by guilhermespopolin — APPROVED** [review]

**Comment by robinandeer** [comment]
@robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2600).

**Comment by robinandeer** [comment]
@robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2600).

**Pull request merged**